Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java

```diff
@@ -47,14 +47,27 @@ public int getReadPageCount() {
     return result;
   }
 
-  public ArrayList<ScriptedURL> getUrlList() {
+  public ArrayList<ScriptedURL> getAllUrlList() {
     SQLiteDatabase dataBase = getReadableDatabase();
     ArrayList<ScriptedURL> resultList = new ArrayList<ScriptedURL>();
 
     Cursor cursor = dataBase.rawQuery("SELECT * from page ", null);
     while (cursor.moveToNext()) {
-      ScriptedURL scriptecItem = new ScriptedURL(cursor.getString(2), cursor.getInt(1));
-      resultList.add(scriptecItem);
+      ScriptedURL scriptedItem = new ScriptedURL(cursor.getString(2), cursor.getInt(1), cursor.getInt(3));
```

Comment:
Please leave those as a comment in the source or using some const integer would be also fine.